docker-compose.yml

```diff
@@ -111,6 +111,37 @@ services:
     networks:
       - io-fn
 
+  functions-admin:
+    image: functions-template:v2.0.0
+    build:
+      context: ./
+      dockerfile: functions/Dockerfile
+    command: ["sh", "-c", "git clone https://github.com/pagopa/io-functions-admin.git /usr/src/app && git checkout $FUNCTIONS_ADMIN_BRANCH && yarn install && yarn build && func extensions install && func start --javascript"]
```

Comment:
I'd rather try to build the generic "functions" container first using `command: /bin/true` in docker-compose
then create a standalone container for every function using `depends_on: functions`


```
# io-functions-admin/Dockerfile
FROM functions

RUN git clone https://github.com/pagopa/io-functions-admin.git /usr/src/app && 
         git checkout $FUNCTIONS_ADMIN_BRANCH && \
         yarn install && \
         yarn build && \
         func extensions install

CMD ["func", "start", "--javascript"]
```

in this way you get a leaner docker-compose and the chance to override any command